[HOTFIX] Add examples to Rules Engine variables (#393)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx

| `${remote_addr}` | The IP address of the client performing the HTTP request. | Request Phase<br />Response Phase |
| `${remote_port}` | The HTTP port being used in the request URL of the client. | Request Phase<br />Response Phase |
| `${remote_user}` | The username provided by basic authentication, if any. | Request Phase<br />Response Phase |
| `${request}` | Original first request line of the client request. It is composed of the request method, the request URI and the HTTP version. For example: `GET /path HTTP/2.0`. | Request Phase<br />Response Phase |
| `${request_method}` | The HTTP method of the request. For example: `GET`. | Request Phase<br />Response Phase |

> | `${request_uri}` | The complete URI of the request, with arguments (query string).  Special UTF-8 characters are URL encoded. For example: `/path?var=value%20of%20var`. | Request Phase<br />Response Phase |

Author: ```suggestion
| `${request_uri}` | The complete URI of the request, with arguments (query string). Special UTF-8 characters are URL encoded. For example: `/path?var=value%20of%20var`. | Request Phase<br />Response Phase |
```

| `${scheme}` | The scheme of the request. For example: `http` or `https`. | Request Phase<br />Response Phase |
| `${sent_http_name}` | The value of the response header field name. The name argument must be converted to lowercase and the hyphens must be converted to underscore. For example: for a header `Content-Length: 9593`, the variable `${sent_http_content_length}` will assume the value `9593`. | Response Phase |
| `${server_addr}` | The IP address of the server that will be receiving the HTTP request. For example: `200.0.0.0` | Request Phase |

> | `${server_port}` | The HTTP port of the server that will be receiving the request. For example: `8080`. | Request Phase |

Reviewer: ```suggestion
| `${server_port}` | The HTTP port of the server that'll be receiving the request. For example: `8080`. | Request Phase |
```

| `${upstream_cookie_name}` | Value of cookie name sent by the origin using the Set-Cookie header field. If many origins are consulted while the request is processed, only cookies from the last origin are stored. | Response Phase |
| `${upstream_http_name}` | Value of the name header field sent by the origin. The name argument must be converted to lowercase and the hyphens must be converted to underscore. If many origins are consulted while the request is processed, only headers from the last origin are stored. For example: `${upstream_http_server}` will assume the value of the Server header field sent by the last queried origin. | Response Phase |
| `${upstream_status}` | Status code of the origin response sent to RTM. If many origins are consulted while the request is processed, the status codes will be separated by a comma (`,`). If an internal redirect from one group of servers to another is initiated by an `X-Accel-Redirect` or an error page, the status codes of the different groups will be separated by a colon (`:`). | Response Phase |

> | `${uri}` | The normalized (URL decoded) URI of the request. For example: `/path/my file.txt`<br /><br />The value of `${uri}` can change during the processing of a request, for example, when an internal redirect occurs or when index files are used. For query string parameters and URL encoded characters, use `${request_uri}`. | Request Phase<br />Response Phase |

Author: Essa vou deixar como estava, o URL decoded e outro termo para `normalized` e não para `URI` - pode ficar confuso haha.

> | `${uri}` | The normalized (URL decoded) URI of the request. For example: `/path/~new/my file.txt`<br /><br />The value of `${uri}` can change during the processing of a request, for example, when an internal redirect occurs or when index files are used. For query string parameters and URL encoded characters, use `${request_uri}`. | Request Phase<br />Response Phase |

Author: ```suggestion
| `${uri}` | The normalized (URL decoded) URI of the request. For example: `/path/~new/my file.txt`. <br /><br />The value of `${uri}` can change during the processing of a request, for example, when an internal redirect occurs or when index files are used. For query string parameters and URL encoded characters, use `${request_uri}`. | Request Phase<br />Response Phase |
```
